pkg/yamltemplate/filetests/if-err-too-many-end.yml

```diff
@@ -12,4 +12,4 @@ metadata:
 ERR: 
 - unexpected end
     stdin:8 |     #@ end
-    stdin:? | __ytt_tplXXX_start_node(11) # *yamlmeta.Document (generated)
\ No newline at end of file
+    stdin:? | __ytt_tplXXX_start_node(11) # document (generated)
```

Comment:
In this case, the reader is _at least_ an advanced ytt user, if not a maintainer. Also, internal details abound in this output. By my eye, it's appropriate to present the Go type, here.

Reply:
Thanks for the review. I have made the code required changes.